CoreApplication/Migrations/Configuration - Copy.txt
namespace Decktra.PubliPuntoEstacion.CoreApplication.Migrations
{
    using Decktra.PubliPuntoEstacion.CoreApplication.Model;
    using System.Data.Entity.Migrations;

    internal sealed class ConfigurationCopy : DbMigrationsConfiguration<Decktra.PubliPuntoEstacion.CoreApplication.Context.PubliPuntoContext>
    {
        public ConfigurationCopy()
        {
            AutomaticMigrationsEnabled = true;
        }

        protected override void Seed(Decktra.PubliPuntoEstacion.CoreApplication.Context.PubliPuntoContext context)
        {
            var ramoComercial = context.RamoComercials.Add(new RamoComercial { Codigo = "001", Nombre = "Comida Rapida" });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "001",
                    Nombre = "Pizza Hut",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    WebAddress = "http://www.pizzahut.net.ve/",
                    Promocion = "Pizza 2 x 1 los miercoles y jueves",
                    ImagenUrl = "pizza01.jpg",
                    LogoUrl = "logo_pizzahut.jpg",
                    VideoUrl = string.Empty
                });

            ramoComercial = ramoComercial = context.RamoComercials.Add(new RamoComercial { Codigo = "002", Nombre = "Seguridad" });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "002",
                    Nombre = "SIESCOM C.A.",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    WebAddress = "http://www.siescom.com.ve/",
                    VideoUrl = "siescom_final.wmv"
                });
        }

        //private void Seed()
        //{
        //    _ramoComercials = new List<RamoComercial>();
        //    var ramoComercial = new RamoComercial
        //    {
        //        Id = 0,
        //        Nombre = "Animales",
        //    };
        //    _ramoComercials.Add(ramoComercial);

        //    _ramoComercials.Add(new RamoComercial { Id = 1, Nombre = "Alimentos" });
        //    _ramoComercials.Add(new RamoComercial { Id = 2, Nombre = "Antiguedades" });
        //    _ramoComercials.Add(new RamoComercial { Id = 3, Nombre = "Bancos" });
        //    _ramoComercials.Add(new RamoComercial { Id = 4, Nombre = "Comida Rapida" });
        //    _ramoComercials.Add(new RamoComercial { Id = 5, Nombre = "Muebles" });
        //    _ramoComercials.Add(new RamoComercial { Id = 6, Nombre = "Peluquerias" });
        //    _ramoComercials.Add(new RamoComercial { Id = 7, Nombre = "Seguridad" });
        //}

        //private void Seed()
        //{
        //    _enteComercials = new List<EnteComercial>();
        //    var ramoComercial = new RamoComercial { Id = 3, Nombre = "Bancos" };

        //    _enteComercials.Add(new EnteComercial { Id = 0, Nombre = "Banco Mercantil", RamoComercial = ramoComercial });
        //    _enteComercials.Add(new EnteComercial { Id = 1, Nombre = "Banesco", RamoComercial = ramoComercial });
        //    _enteComercials.Add(new EnteComercial { Id = 2, Nombre = "Banco Provincial", RamoComercial = ramoComercial });

        //    ramoComercial = new RamoComercial { Id = 4, Nombre = "Comida Rapida" };
        //    _enteComercials.Add(new EnteComercial
        //    {
        //        Id = 3,
        //        Nombre = "Pizza Hut",
        //        RamoComercial = ramoComercial,
        //Rif = "J-123-456-789",
        //Telefonos = "0251.251.11.11 - 0416.111.11.11",
        //Direccion = "Av. Lara con Av. Leones, Local 05, al lado del Burguer King",
        //WebAddress = "http://www.pizzahut.net.ve/",
        //Promocion = "Pizza 2 x 1 los miercoles y jueves",
        //ImagenUrl = "pizza01.jpg",
        //LogoUrl = "logo_pizzahut.jpg",
        //VideoUrl = string.Empty
        //    });
        //    _enteComercials.Add(new EnteComercial
        //    {
        //        Id = 4,
        //        Nombre = "Mac Donald",
        //        RamoComercial = ramoComercial,
        //        VideoUrl = "algo.wmv"
        //    });

        //    ramoComercial = new RamoComercial { Id = 7, Nombre = "Seguridad" };
        //    _enteComercials.Add(new EnteComercial
        //    {
        //        Id = 5,
        //        Nombre = "SIESCOM C.A.",
        //        RamoComercial = ramoComercial,
        //        VideoUrl = "videos/siescom_final.wmv"
        //    });
        //}
    }
}


CoreApplication/Migrations/Configuration.cs
namespace Decktra.PubliPuntoEstacion.CoreApplication.Migrations
{
    using Decktra.PubliPuntoEstacion.CoreApplication.Model;
    using System.Data.Entity.Migrations;

    internal sealed class Configuration : DbMigrationsConfiguration<Decktra.PubliPuntoEstacion.CoreApplication.Context.PubliPuntoContext>
    {
        public Configuration()
        {
            AutomaticMigrationsEnabled = true;
            AutomaticMigrationDataLossAllowed = true;
        }

        protected override void Seed(Decktra.PubliPuntoEstacion.CoreApplication.Context.PubliPuntoContext context)
        {
            var ramoComercial = context.RamoComercials.Add(new RamoComercial { Codigo = "CR", Nombre = "Comida Rapida" });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "CR-001",
                    Nombre = "Pizza Hut",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    WebAddress = "http://www.pizzahut.net.ve/",
                    Promocion = "Pizza 2 x 1 los miercoles y jueves",
                    ImagenUrl = "pizza01.jpg",
                    LogoUrl = "logo_pizzahut.jpg"
                });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "CR-002",
                    Nombre = "Spigolo Pizza",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    WebAddress = "http://www.spigolopizza.com.ve/",
                    LogoUrl = "ejemp_empre2.jpg"
                });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "CR-002",
                    Nombre = "Pizza Cono",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    WebAddress = "http://www.pizzacono.com.ve/",
                    LogoUrl = "ejemp_empre1.jpg"
                });

            //---------------------------

            ramoComercial = ramoComercial = context.RamoComercials.Add(new RamoComercial { Codigo = "SCT", Nombre = "Seguridad" });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "SCT-001",
                    Nombre = "SIESCOM C.A.",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    WebAddress = "http://www.siescom.com.ve/",
                    VideoUrl = "siescom_final.wmv",
                    ImagenUrl = "siescom_gra.jpg",
                    LogoUrl = "siescom_peq.jpg"
                });

            //---------------------------

            ramoComercial = ramoComercial = context.RamoComercials.Add(new RamoComercial { Codigo = "DCT", Nombre = "Discotecas" });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "DCT-001",
                    Nombre = "Dance Club",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    WebAddress = "http://www.danceclub.com.ve/",
                    ImagenUrl = "danceclub_gra.jpg",
                    LogoUrl = "danceclub_peq.jpg"
                });

            //---------------------------

            ramoComercial = ramoComercial = context.RamoComercials.Add(new RamoComercial { Codigo = "ZPT", Nombre = "Zapaterias" });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "ZPT-001",
                    Nombre = "DC Shoes",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    WebAddress = "http://www.dcshoes.com.ve/",
                    ImagenUrl = "dcshoes_gra.jpg",
                    LogoUrl = "dcshoes_peq.jpg"
                });

            //---------------------------

            ramoComercial = ramoComercial = context.RamoComercials.Add(new RamoComercial { Codigo = "RST", Nombre = "Restaurant" });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "RST-001",
                    Nombre = "Bellavino",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    WebAddress = "http://www.bellavino.com.ve/",
                    LogoUrl = "ejemp_empre3.jpg"
                });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "RST-002",
                    Nombre = "Restaurant",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    WebAddress = "http://www.restaurant.com.ve/",
                    ImagenUrl = "restaurant_gra.jpg",
                    LogoUrl = "restaurant_peq.jpg"
                });

            //---------------------------

            ramoComercial = ramoComercial = context.RamoComercials.Add(new RamoComercial { Codigo = "PBC", Nombre = "Publicidad" });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "PBC-001",
                    Nombre = "Gabriel Anzola",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    WebAddress = "gabrielanzola@gmail.com",
                    ImagenUrl = "gabrielanzola_gra.jpg",
                    LogoUrl = "grabrielanzola_peq.jpg"
                });

            //---------------------------

            ramoComercial = ramoComercial = context.RamoComercials.Add(new RamoComercial { Codigo = "HTL", Nombre = "Hoteles" });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "HTL-001",
                    Nombre = "Hotel",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    WebAddress = "http://www.hotel.com.ve/",
                    ImagenUrl = "hotel_gra.jpg",
                    LogoUrl = "hotel_peq.jpg"
                });

            //---------------------------

            ramoComercial = ramoComercial = context.RamoComercials.Add(new RamoComercial { Codigo = "TLL", Nombre = "Talleres" });

            context.EnteComercials.AddOrUpdate(q => q.Codigo,
                new EnteComercial
                {
                    Codigo = "TLL-001",
                    Nombre = "Todo Taller",
                    RamoComercial = ramoComercial,
                    Rif = "J-123-456-789",
                    Direccion = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
                    Telefonos = "0251.251.11.11 - 0416.111.11.11",
                    WebAddress = "http://www.taller.com.ve/",
                    ImagenUrl = "taler_gra.jpg",
                    LogoUrl = "taler_peq.jpg"
                });
        }
    }
}
